src/frontends/tensorflow/src/op/approximate_equal.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/abs.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/select.hpp"
+#include "openvino/op/subtract.hpp"
+
+using namespace std;
+using namespace ov;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_approximate_equal_op(const NodeContext& node) {
+    auto op_type = node.get_op_type();
+    TENSORFLOW_OP_VALIDATION(node, op_type == "ApproximateEqual", "Incorrect usage of translate_approximate_equal_op.");
+
+    // Extract necessary attributes and inputs
+    auto tolerance = node.get_attribute<float>("tolerance");
```

Comment:
please set default value as well 1e-05

```suggestion
    auto tolerance = node.get_attribute<float>("tolerance", 1e-5f);
```